### DISCUSSION TITLE
【求助】alist挂载在路由器上，然后怎么把路由器下的电脑的目录挂载进来

### DISCUSSION THREAD

**Discussion by XiaoYinP** [AlistGo/alist#4060] [General]
alist挂载在电脑上就有电脑要关机的烦恼，挂载在路由器上就没有便捷的访问电脑文件的烦恼，各位大佬们有什么解决的方法吗

**Comment by Rektyfikowany** [comment]
电脑开机的时候可以也跑个alist或其它webdav或SMB服务供路由器挂载，关机的话考虑NAS

**Reply by XiaoYinP** [reply]
可以不通过webdev直接把电脑的alist挂载到路由器上的alist吗，省一个电脑进程（

**Reply by Rektyfikowany** [reply]
alist之间挂载可以参考[guide/drivers/AlistV2V3](https://alist.nn.ci/zh/guide/drivers/Alist%20V2%20V3.html)，需要电脑和路由器各自运行alist。不想单开进程可以根据你的操作系统选预装的网络存储，windows下可考虑SMB，相当于是系统后台自动管理一个进程

**Reply by XiaoYinP** [reply]
我电脑开启alist，路由器用alistv3协议进行对电脑的alist挂载的时候，链接填的是电脑的ipv4地址，链接加上端口号会提示链接不能有冒号，去掉端口会显示不支持的协议（电脑使用的确实是v3版本）。SMB速度会比较慢，不是很实用

**Reply by Rektyfikowany** [reply]
我测试的挂载`http://192.168.1.2:5244`访问正常，提示不能有冒号会不会是填写格式的问题引起解析失败

**Reply by XiaoYinP** [reply]
可以了，好像是因为我没有加上http://这个协议头的问题，非常感谢你的指导
